src/librustc/diagnostics.rs

```diff
@@ -112,6 +112,11 @@ reference when using guards or refactor the entire expression, perhaps by
 putting the condition inside the body of the arm.
 "##,
 
+E0018: r##"
+The value of static and const variables must be known at the compile-time which is
```

Comment:
He means "... value of static and const variables must be known at compile time, which is ...".
And I think he's right.